tonic-web/src/call.rs

```diff
@@ -305,7 +305,11 @@ where
                     }
                     FindTrailers::IncompleteBuf => continue,
                     FindTrailers::Done(len) => {
-                        Poll::Ready(Some(Ok(Frame::data(buf.split_to(len).freeze()))))
+                        if len > 0 {
```

Comment:
Nit: please spell it like this:

```rust
Poll::Ready(match len {
    0 => None,
    _ => Some(Ok(Frame::data(buf.split_to(len).freeze()))),
})
```

Reply:
good idea, done.

Reply:
cargo fmt ended up inlining this expression with `FindTrailers::Done(len) =>`, which is neat enough I think